data/common/include/PercivalEmulatorDefinitions.h

```diff
@@ -5,33 +5,20 @@
  *      Author: tcn45
  */
 
-#ifndef INCLUDE_PERCIVALEMULATORDEFINITIONS_H_
-#define INCLUDE_PERCIVALEMULATORDEFINITIONS_H_
-
-#define P2M_EMULATOR_NEW_FIRMWARE 1
+#pragma once
 
 namespace PercivalEmulator {
 
-#ifdef P2M_EMULATOR_NEW_FIRMWARE
-
+    // rename this because it's the datablock size
```

Comment:
As you have removed the emulator definitions from this file I wonder if it is time to rename the file itself so that it isn't confusing?  Maybe something like PercivalDefinitions.h?

Reply:
yes ok but I haven't renamed the plugin as that is a bridge too far ??!